packages/bpk-docs/src/pages/ThemingPage/ThemingPage.js

```diff
@@ -159,10 +159,10 @@ const components = [
 
 const blurb = [
   <Paragraph>
-    Backpack has full theming support through its theme provider component. This
-    component provides a theme to all themeable components underneath itself. In
-    the render tree, all themeable components will have access to the provided
-    theme, even when they are multiple levels deep.
+    All Backpack components have full theming support through through the theme
+    provider component. In the render tree, all themeable components will have
+    access to the provided theme, even when they are multiple levels deep.
+    variables are used.
```

Comment:
> even when they are multiple levels deep. variables are used.

`variables are used.` doesn't need to be here.

Reply:
There's the 'the following web/native components are themeable' bit, I think we can remove that now as it's just all components that are themeable, plus we've not kept the list up-to-date.